### ISSUE TITLE
How can we do Hacktoberfest?!

### ISSUE THREAD

**Issue by mikesplain** [kubernetes/kops#9920]
As mentioned during office hours & other locations, lets start discussing how we can participate in [Hacktoberfest](https://github.com/topics/hacktoberfest)!

Here are our currently tagged [Good first issue](https://github.com/kubernetes/kops/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)

**Comment by olemarkus** [comment]
https://github.com/kubernetes/kops/labels/lifecycle%2Ffrozen may also contain something worth adding.

**Comment by dvdmuckle** [comment]
Looking over issues with the "good first issue" tag, some felt hard to parse as someone new to the project. As someone looking to contribute for the first time, it would help, for issues that will be up for grabs for Hacktoberfest, that maybe someone familiar with the issue could sign up to be a mentor in that issue, so someone looking to contribute could ask both questions about that issue and broader questions related to contributing such as testing, code quality, etc. Alternatively, Hacktoberfest suggests holding hackathon style events, maybe instead a weekly office hour meeting could be set for anyone with questions.

**Comment by justinsb** [comment]
I'm going to suggest that we do hacktoberfest - I'm happy to try to lead office hour type events at least once a week.  I also suggest we can use these time slots to explore alternative time slots (#9833)

**Comment by dezmodue** [comment]
Hello maintainers, the hacktoberfest rules have changed with the aim of making the event opt-in for projects and reduce the amount of "spammy" contributions - blog post [here](https://hacktoberfest.digitalocean.com/hacktoberfest-update).

In order to opt in the project needs to add a topic 'hacktoberfest' to the repository and in order for a PR to "count" towards hacktoberfest it needs to be:

Submitted in a repo with the hacktoberfest topic AND
during the month of October AND (
  The PR is merged OR
  The PR is labelled as hacktoberfest-accepted by a maintainer OR
  The PR has been approved
)

**Comment by olemarkus** [comment]
/remove-lifecycle rotten
/close

Until next year :)
